Finmer.Game/Gameplay/SceneCombat2.cs

```diff
@@ -314,8 +325,8 @@ private void HandlePlayerInputDefault(ECombatAction action)
             {
                 case ECombatAction.Attack:
                 case ECombatAction.GrappleInitiate:
```

Comment:
It appears that this is the issue: `Attack` and `GrappleInitiate` share the same code, so if you attack it will now get a list of grapple targets and try to start a grapple even though you actually clicked the Attack button. The two case labels should be separated, instead, like Swallow does it.